qemu/tests/x86_cpu_model.py

```diff
@@ -19,11 +21,18 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment.
     """
+    qemu_binary = utils_misc.get_qemu_binary(params)
```

Comment:
out = VM.cmd(""query-cpu-definitions")   is enough.

Reply:
IIUC, vm.cmd() is used to run cmd inside guest. Instead, we need to send a qmp cmd with '-M none' in qemu cli. Details pls see bug1786257. Thanks.

Reply:
> IIUC, vm.cmd() is used to run cmd inside guest. Instead, we need to send a qmp cmd with '-M none' in qemu cli. Details pls see bug1786257. Thanks.

sorry, vm.minitor.cmd()

Reply:
Since we don't boot vm before query cpu anymore, we don't have a vm instance at the beginning.